sre-cloud.md

```diff
@@ -0,0 +1,64 @@
+# S.R.E / Cloud Challenge
+
+## About the Challenge... 
+
+We have two challenges, the first one is an intermediate test, and the second is a more advanced: The both challenges is about defining an architecture for Zé at AWS, considering several requirements that are necessary to consider.
```

Comment:
Linha 5:
Só fui perceber agora esse "The both challenges" (plural) "... is about" (singular). Corrigir concordância verbal.

"[...] considering several requirements that are necessary to consider"
Considerando os requisitos que são necessários de considerar? Ruim essa frase né? Só vi agora também.